worlds/metroidprime/Items.py

```diff
@@ -1,63 +1,81 @@
 from BaseClasses import Item, ItemClassification
-import typing
 
+AP_METROID_PRIME_ITEM_ID_BASE = 5031000
```

Comment:
Tell me if you want me to flip this back to how this was previously 👍 I figured splitting it out this way would make it a little cleaner on the client for interacting with item ids and archipelago ids separately while ensuring the archipelago id is still generated correctly